### DISCUSSION TITLE
Update 'docker-compose.override.yml' in HuggingFace?

### DISCUSSION THREAD

**Discussion by ByungsunAhn** [danny-avila/LibreChat#4238] [Q&A] [answered]
See below link, in #Custom Configuration - Hugging Face, number 2.
(https://www.librechat.ai/docs/configuration/rag_api#custom-configuration---hugging-face)

How to update 'docker-compose.override.yml' in HuggingFace?

Only files I see in huggingface Space are, 
.gitattributes,
Dockerfile
README.md
librechat.yaml.

Do I have to manually add docker-compose.override.yml?

Thanks in advance.

**Comment by fuegovic** [comment, marked as answer]
huggingface doesn't work the same way, there's no override file. See this to add rag on HF:
https://github.com/danny-avila/LibreChat/issues/4062#issuecomment-2354527243

**Reply by ByungsunAhn** [reply]
Thank you. 
I duplicated your Space and setup a PGRESQL in supabase. Rag Space runs well.
However, I don't know how to connect LibreChat with Rag.
What do I input in RAG_API_URL, and is it enough?
